Dockerfile.test.cpu

```diff
@@ -8,9 +8,9 @@ ARG MPI_KIND=OpenMPI
 ARG PYTHON_VERSION=2.7
 ARG TENSORFLOW_PACKAGE=tensorflow==1.14.0
 ARG KERAS_PACKAGE=keras==2.2.4
-ARG PYTORCH_PACKAGE=torch==1.1.0
-ARG TORCHVISION_PACKAGE=https://download.pytorch.org/whl/cpu/torchvision-0.3.0-cp27-cp27mu-linux_x86_64.whl
-ARG MXNET_PACKAGE=mxnet==1.4.1
+ARG PYTORCH_PACKAGE=torch==1.2.0+cpu
+ARG TORCHVISION_PACKAGE=torchvision==0.4.0+cpu
+ARG MXNET_PACKAGE=mxnet==1.5.0
```

Comment:
Intending to bump MXNet version here as well?

Reply:
Nevermind. See that MXNet was supposed to be bumped to v1.5 in #1251. Good catch.